pegasus/sites.v3/code.org/views/accelerated_next.haml

```diff
@@ -1,10 +1,8 @@
-%h1 What's Next?
+%h2 Code.org highly recommends these third party applications:
 
-%h3 Code.org highly recommends these third party applications:  
-
-%div{:style=>"background:#fff5e0; border-radius:10px; float:none; height:360px; padding:15px;"}
+.thirdparty-box
   %a{:href=>"https://studio.code.org/levels/2621?show_callouts=true"}
-    %img{:src=>"https://images.code.org/9df60110a2abe207c581bd35ac2187e3-image-1433886030096.png", :style=>"float:left; width:300px; margin-right:20px; border-radius:10px;"}
+    %img.thirdparty-image{:src=>"https://images.code.org/9df60110a2abe207c581bd35ac2187e3-image-1433886030096.png"}
```

Comment:
Yeah... this was Kiki's content stuff. Good idea, I'll just download and add them to our system.